public/actions.js

```diff
@@ -605,6 +605,45 @@ export const StartTimer = (state, { timerStartedAt, timerDuration }) => [
   }),
 ];
 
+export const Skip = (state, { timerDuration, timerStartedAt = null }) => {
+  console.log("action - Next");
+  if (state.mob.length === 0) {
+    return state;
+  }
+
+  if (!timerStartedAt) {
+    timerStartedAt = Date.now();
+  }
+
+  const effectsToRun = [
+    effects.CompleteTimer({
+      websocket: state.websocket,
+    }),
+    effects.andThen({
+      action: CycleMob, // eslint-disable-line no-use-before-define
+      props: { },
+    }),
+    effects.andThen({
```

Comment:
For now, I think we'll remove `StartTimer` (and the timerStartedAt stuff). I'm willing to be wrong and have a larger discussion on it, though.

Reply:
I'm good with that, however that leaves a few other questions to be answered

First, when Skip (or whatever we call it) is clicked does it only cycle or does it end the turn and cycle?

Second, if we land on the semantics that skip will only cycle the mob, and possibly end the existing session, if one is active, does the skip button appear all of the time? This would result in being able to cycle people before the turn.

Finally, if the decision is to leave the button all of the time, is there a better location for the button? Two thoughts I had:
- Place the button else where
- Place skip to left of Start, this way start/pause interchange with one another and if someone double clicks it only pauses and doesn't accidentally skip. I was thinking we might wish to make a similar change either way.

Reply:
Functionally, I think we can always stop the timer. If there is no timer running, then that should essentially be a no-op, so there shouldn't be any harm.

In terms of placement, I'm not picky. Move it to a few sane places, and take screenshots and let's take a peek 👀